p2p/src/simulated/ingress.rs

```diff
@@ -54,29 +64,12 @@ impl Oracle {
         &mut self,
         public_key: PublicKey,
         channel: Channel,
-        max_size: usize,
     ) -> Result<(Sender, Receiver), Error> {
         let (sender, receiver) = oneshot::channel();
         self.sender
             .send(Message::Register {
                 public_key,
                 channel,
-                max_size,
-                result: sender,
-            })
-            .await
-            .map_err(|_| Error::NetworkClosed)?;
-        receiver.await.map_err(|_| Error::NetworkClosed)?
-    }
-
-    /// Deregister a peer from the network.
-    ///
-    /// If the peer is not registered, this will return an error.
-    pub async fn deregister(&mut self, public_key: PublicKey) -> Result<(), Error> {
```

Comment:
As discussed, this is unused and its intended function is better served with `remove_link`. Good to remove.